packages/ra-supabase/README.md

```diff
@@ -62,15 +55,116 @@ export const authProvider = supabaseAuthProvider(supabase, {
 
 // in App.js
 import { Admin, Resource, ListGuesser } from 'react-admin';
-import { authRoutes } from 'ra-supabase';
+import { LoginPage } from 'ra-supabase';
 import { dataProvider } from './dataProvider';
 import { authProvider } from './authProvider';
 
 export const MyAdmin = () => (
     <Admin
         dataProvider={dataProvider}
         authProvider={authProvider}
-        customRoutes={authRoutes}
+        loginPage={LoginPage}
+    >
+        <Resource name="posts" list={ListGuesser} />
+        <Resource name="authors" list={ListGuesser} />
+    </Admin>
+);
+```
+
+## Features
+
+### DataProvider
+
+`ra-supabase` is built on [`ra-data-postgrest`](https://github.com/raphiniert-com/ra-data-postgrest/tree/v2.0.0-alpha.0) that leverages [PostgREST](https://postgrest.org/en/stable/). As such, you have access the following features:
```

Comment:
Many sections are duplicated in the documentation files. Isn't there a better way to organize the docs so that the actual doc lives at only one place (making it easier to maintain it), and the other packages would simply offer a link to that doc ?

Reply:
Agreed

Reply:
It's better. You still have some duplicate sections between `packages/ra-supabase/README.md` and `packages/ra-supabase-core/README.md`.

To me, most of the docs should be in `ra-supabase` since that's what most people will download, and most of them will only look a the docs from the npmjs website.

What do you think?

Reply:
I think duplication between those two is actually expected. Most people will probably use `ra-supabase` but those who use only `ra-supabase-core` should also have its documentation ih the readme